### ISSUE TITLE
RunPod not showing Jupyter Lab option

### ISSUE THREAD

**Issue by hugo-k-m** [TheLastBen/fast-stable-diffusion#2887]
The runpod template isn't providing the option connect to jupyter lab anymore.

![Screenshot 2024-07-18 162328](https://github.com/user-attachments/assets/7cef7b74-3e5a-4f7b-85d2-c23de5f381be)

**Comment by TheLastBen** [comment]
Runpd decided to discontinue support for fast stable diffusion notebooks, it's out of my hand

**Comment by hugo-k-m** [comment]
@TheLastBen oh I see. That's unfortunate.

Is colab the only option remaining then? Since paperspace seems to have issues with this as well.

**Comment by hcatGitHub** [comment]
<img width="602" alt="Screenshot 2024-07-18 at 16 48 57" src="https://github.com/user-attachments/assets/874bb4dc-c21f-478f-a367-4fa4091f2a8a">
It is still there, really discontinued?  I have been using it for the whole day until now.  It was suddenly disconnected and when I try to connect again.  I got an error when installing the dependencies.

Traceback (most recent call last):

  File /usr/local/lib/python3.10/dist-packages/IPython/core/interactiveshell.py:3508 in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)

  Cell In[1], line 8
    with open('/dev/null', 'w') as devnull:import requests, os, time, importlib;open('/workspace/mainrunpodA1111.py', 'wb').write(requests.get('https://huggingface.co/datasets/TheLastBen/RNPD/raw/main/Scripts/mainrunpodA1111.py').content);os.chdir('/workspace');time.sleep(2);import mainrunpodA1111;importlib.reload(mainrunpodA1111);from mainrunpodA1111 import *;Deps(force_reinstall)

  File /workspace/mainrunpodA1111.py:1
    Invalid username or password.
            ^
SyntaxError: invalid syntax

I pay Runpod only to use this template.  If they discontinue supprt for it, I will contact them try to work out why.

**Comment by RONNYKHALIL** [comment]
> I pay Runpod only to use this template. If they discontinue supprt for it, I will contact them try to work out why.

same here. I just finished fully migrating from paperspace and settling in, depending on runpod+fast-stable-diffusion for the foreseeable future. I just wrote to runpod and let them know all my business with them depends on support for this template. it sucks cause last week just rented a 4000 pod for a month

**Comment by TheLastBen** [comment]
> @TheLastBen oh I see. That's unfortunate.
> 
> Is colab the only option remaining then? Since paperspace seems to have issues with this as well.

Paperspace is up and running

**Comment by RONNYKHALIL** [comment]
> > @TheLastBen oh I see. That's unfortunate.
> > Is colab the only option remaining then? Since paperspace seems to have issues with this as well.
> 
> Paperspace is up and running

say whaaa. ok now theyre just fuckin with me lol

**Comment by HelpANewbie** [comment]
@TheLastBen 

Is there any chance that Runpod will start it again ? Should I ask them for a refund and start with Paperspace ? 

Anyone knows about how pricing works on Paperspace for using SD ?

**Comment by kodxana** [comment]
I mean template would still work if it would not relay on external hugging face repo. In that case it's more like breaking it on purpose.

**Comment by dykim723** [comment]
@TheLastBen  I need RNPD mainrunpodDreamboothv1.py.
Can I use dream booth nothbook the same way in paperspace?

**Comment by qudabear** [comment]
So who is going to refund me for the shitty code, the broken disconnects, the terrible training results, and now the random disconnect of the service I was using?

**Comment by qudabear** [comment]
I highly suggest everyone on this thread put pressure on Runpod to restore the service, or demand refunds. I also highly suggest Ben improves his code and provides detailed settings information for better results. This is absolute nonsense, and we've all been scammed.

**Comment by TheLastBen** [comment]
> @TheLastBen I need RNPD mainrunpodDreamboothv1.py. Can I use dream booth nothbook the same way in paperspace?

Yes

**Comment by ashleykleynhans** [comment]
> I highly suggest everyone on this thread put pressure on Runpod to restore the service, or demand refunds. I also highly suggest Ben improves his code and provides detailed settings information for better results. This is absolute nonsense, and we've all been scammed.

1. RunPod did not remove the service, TheLastBen decided to break it to be spiteful after RunPod decided to stop sponsoring him, but I guess that was a poor call by RunPod considering that they just recently received a significant amount of investment.
2. Dreambooth is unpredictable and mostly depends on the quality of your training dataset.  Nobody can give you settings that will guarantee better results.  You have to run multiple experiments and compare them to find the best results.

**Comment by qudabear** [comment]
> > I highly suggest everyone on this thread put pressure on Runpod to restore the service, or demand refunds. I also highly suggest Ben improves his code and provides detailed settings information for better results. This is absolute nonsense, and we've all been scammed.
> 
>     1. RunPod did not remove the service, TheLastBen decided to break it to be spiteful after RunPod decided to stop sponsoring him, but I guess that was a poor call by RunPod considering that they just recently received a significant amount of investment.
> 
>     2. Dreambooth is unpredictable and mostly depends on the quality of your training dataset.  Nobody can give you settings that will guarantee better results.  You have to run multiple experiments and compare them to find the best results.

I had a sneaking suspicion it was something like that. As for #2, I'm intimately aware how Dreambooth works, and this is, in fact, false. Assuming a properly prepared dataset, there are a range of settings that are good practice, and many that are not worth trying, objectively.  If Ben's repo was as flexible and amazing as he claims, there would be a consistent range of settings that would work with it, but there really aren't - unless you are only training faces or celebrities, with few photos. He's been asked directly about these details, especially regarding how the text encoder works, and he's dodged the question every time. This is a feature of him, not Dreambooth.

**Comment by fireincairo00** [comment]
I am really confused that Ben said he removed the Colab support because it was unstable. IMO Colab is one of the most stable options out there lol. Hope he can re-consider his decisions - ppl will move to other devs

**Comment by TheLastBen** [comment]
> > I highly suggest everyone on this thread put pressure on Runpod to restore the service, or demand refunds. I also highly suggest Ben improves his code and provides detailed settings information for better results. This is absolute nonsense, and we've all been scammed.
> 
> 1. RunPod did not remove the service, TheLastBen decided to break it to be spiteful after RunPod decided to stop sponsoring him, but I guess that was a poor call by RunPod considering that they just recently received a significant amount of investment.
> 2. Dreambooth is unpredictable and mostly depends on the quality of your training dataset.  Nobody can give you settings that will guarantee better results.  You have to run multiple experiments and compare them to find the best results.

The notebooks were under a contract since the beginning, not sponsorship, the contract ended, the service stops, no spite whatsoever, what happens when your runpod balance goes to zero ? does their service stop out of spite ? do you expect the service to continue for free ? and save 25$/day ?

**Comment by ashleykleynhans** [comment]
> > > I highly suggest everyone on this thread put pressure on Runpod to restore the service, or demand refunds. I also highly suggest Ben improves his code and provides detailed settings information for better results. This is absolute nonsense, and we've all been scammed.
> > 
> > 
> > 
> > 1. RunPod did not remove the service, TheLastBen decided to break it to be spiteful after RunPod decided to stop sponsoring him, but I guess that was a poor call by RunPod considering that they just recently received a significant amount of investment.
> > 2. Dreambooth is unpredictable and mostly depends on the quality of your training dataset.  Nobody can give you settings that will guarantee better results.  You have to run multiple experiments and compare them to find the best results.
> 
> The notebooks were under a contract since the beginning, not sponsorship, the contract ended, the service stops, no spite whatsoever, what happens when your runpod balance goes to zero ? does their service stop out of spite ? do you expect the service to continue for free ? and save 25$/day ?

Okay that makes some sense, but doesn't make sense as to why they wouldn't renew the contract since so many people depend on the template.  Using your anaology, if your balance runs out, you add more, you don't just shrug it off and basically say screw you to the users.

**Comment by TheLastBen** [comment]
> > > >
> Okay that makes some sense, but doesn't make sense as to why they wouldn't renew the contract since so many people depend on the template. Using your anaology, if your balance runs out, you add more, you don't just shrug it off and basically say screw you to the users.

I explicitly told them that it would be a great pleasure continuing to work with them, but eventually, they felt it could be replaced and the users will eventually find a way to copy it and use it for free as a community template.

**Comment by TheLastBen** [comment]
> > > > 
> Ben's (Runpod) notebook has been broken or subpar for a long time and it never got fixed.

what was broken about the notebooks ?

**Comment by ashleykleynhans** [comment]
> I explicitly told them that it would be a great pleasure continuing to work with them, but eventually, they felt it could be replaced and the users will eventually find a way to copy it and use it for free as a community template.

Good luck with that, the template is/was extremely complex pulling in files and resources from various different sources.  
 tried to take some time to understand the layout about a year ago, but just gave up with it.  I probably have the most Community templates on RunPod, so I honestly don't see how someone else is going to figure it out either.

**Comment by AlfaShot111** [comment]
There was a code:
Args="--listen --port 3000 --preview-method auto"

#--------------------
sd()
!python /workspace/ComfyUI/main.py $Args

Could you please write what was inside sd()?

**Comment by jockisen** [comment]
> 2. Dreambooth is unpredictable and mostly depends on the quality of your training dataset.  Nobody can give you settings that will guarantee better results.  You have to run multiple experiments and compare them to find the best results.

TheLastBen’s trainer is awesome and not unpredictable at all. I have used it for all my finetunings including Aether Cloud.

**Comment by jockisen** [comment]
> > @TheLastBen oh I see. That's unfortunate.
> > Is colab the only option remaining then? Since paperspace seems to have issues with this as well.
> 
> Paperspace is up and running

<img width="455" alt="Skärmavbild 2024-07-21 kl  21 22 43" src="https://github.com/user-attachments/assets/d09b72b1-b785-4d9c-b566-df522971d3b4">

No idea how to get this working even on paid alternatives.

**Comment by TheLastBen** [comment]
@jockisen did you try a different machine (gpu) ?

**Comment by jockisen** [comment]
> @jockisen did you try a different machine (gpu) ?

I don’t even know how to do that. The readme your provided seems outdated. You need a paid sub + pay like 3x the amount per gpu hour?

**Comment by qudabear** [comment]
> > > > >
> > 
> > 
> > Ben's (Runpod) notebook has been broken or subpar for a long time and it never got fixed.
> 
> what was broken about the notebooks ?

I commented about this and sent you messages on various forums multiple times. Other people have created issues up to a year ago. One issue out of many was that it would constantly give you the "port" error. Adding the port to the pod does nothing to resolve this. The other issue was any settings beyond about 20 images resulted in garbage, and specific questions about text encoder values were never answered. In general, it gives worse results than other repos, but got no response from you. As far as the "contract" issue, I don't think anyone really cares. You could have continued to provide it on Runpod for free.

**Comment by jockisen** [comment]
@TheLastBen yeah man, please fix this and provide the trainer for free. I can’t work without it.

**Comment by jockisen** [comment]
> > > > > >
> > > 
> > > 
> > > Ben's (Runpod) notebook has been broken or subpar for a long time and it never got fixed.
> > 
> > 
> > what was broken about the notebooks ?
> 
> I commented about this and sent you messages on various forums multiple times. Other people have created issues up to a year ago. One issue out of many was that it would constantly give you the "port" error. Adding the port to the pod does nothing to resolve this. The other issue was any settings beyond about 20 images resulted in garbage, and specific questions about text encoder values were never answered. In general, it gives worse results than other repos, but got no response from you. As far as the "contract" issue, I don't think anyone really cares. You could have continued to provide it on Runpod for free.

Have you figured out how to run it on Paperspace?

**Comment by jockisen** [comment]
<img width="397" alt="Skärmavbild 2024-07-22 kl  12 21 36" src="https://github.com/user-attachments/assets/e3fe7386-50ab-4023-b168-86c6ba32a82e">


> @jockisen did you try a different machine (gpu) ?

<img width="397" alt="Skärmavbild 2024-07-22 kl  12 21 36" src="https://github.com/user-attachments/assets/5ec51fc4-3840-43cf-bcad-20afdc59f210">

pulled it from there also now?

**Comment by qudabear** [comment]
> > > > > > >
> > > > 
> > > > 
> > > > Ben's (Runpod) notebook has been broken or subpar for a long time and it never got fixed.
> > > 
> > > 
> > > what was broken about the notebooks ?
> > 
> > 
> > I commented about this and sent you messages on various forums multiple times. Other people have created issues up to a year ago. One issue out of many was that it would constantly give you the "port" error. Adding the port to the pod does nothing to resolve this. The other issue was any settings beyond about 20 images resulted in garbage, and specific questions about text encoder values were never answered. In general, it gives worse results than other repos, but got no response from you. As far as the "contract" issue, I don't think anyone really cares. You could have continued to provide it on Runpod for free.
> 
> Have you figured out how to run it on Paperspace?

No, I gave up on it and am working with Everydream 2 now to try to fix my model.

**Comment by TheLastBen** [comment]
@jockisen simply start a pytorch template, then use https://github.com/TheLastBen/PPS as the starting repo, the notebooks work exactly like runpod

**Comment by jockisen** [comment]
> @jockisen simply start a pytorch template in paperspace gradient, then use https://github.com/TheLastBen/PPS as the starting repo, the notebooks work exactly like runpod

So you just won't let us use your notebook on Runpod anymore? It's not like they've not let you share it for free, right? Even though it messes up with people's work and such? You didn't even care for to inform your users that that was gonna happen?

**Comment by jockisen** [comment]
> @jockisen simply start a pytorch template in paperspace gradient, then use https://github.com/TheLastBen/PPS as the starting repo, the notebooks work exactly like runpod

btw, this information is outdated.

![Uploading Skärmavbild 2024-07-22 kl. 22.47.34.png…]()

that's not how it looks on Paperspace anymore when you click to create a notebook:

<img width="616" alt="Skärmavbild 2024-07-22 kl  22 54 53" src="https://github.com/user-attachments/assets/0775c5f6-f983-4af9-bbe6-ebe65a85ba1f">

**Comment by jockisen** [comment]
@TheLastBen 

<img width="958" alt="Skärmavbild 2024-07-23 kl  00 11 16" src="https://github.com/user-attachments/assets/ee7d0a16-1e6f-455c-9fc3-927d9544a547">

this is how slow it works on the cheapest ($ 1.10/h) 24 gb vram gpu on Paperspace with the exact same settings I've used on Runpod. this is for a 30 image dataset. wouldve taken me like 15 minutes tops on Runpod. 

I suppose I will have to look for some other solution to train after just having used your stuff for two years. I'm very unhappy with this.

**Comment by qudabear** [comment]
> @TheLastBen
> <img alt="Skärmavbild 2024-07-23 kl 00 11 16" width="958" src="https://private-user-images.githubusercontent.com/112794049/351127119-ee7d0a16-1e6f-455c-9fc3-927d9544a547.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE3MDAxNDAsIm5iZiI6MTcyMTY5OTg0MCwicGF0aCI6Ii8xMTI3OTQwNDkvMzUxMTI3MTE5LWVlN2QwYTE2LTFlNmYtNDU1Yy05ZmMzLTkyN2Q5NTQ0YTU0Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyM1QwMTU3MjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MmRhOGVjMmVkZGQ5OGMyMjRmZTU1Yjk4ZjMxNzg4ZjMwNmMxM2EzMmE3Mjc1MTA2NzNlYjE4OGRiNzAwZDAxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.W1BZ-YY-A8XpzgIfYhznKLD2mKc8TSxuGHTFFjGPjfk">
> 
> this is how slow it works on the cheapest ($ 1.10/h) 24 gb vram gpu on Paperspace with the exact same settings I've used on Runpod. this is for a 30 image dataset. wouldve taken me like 15 minutes tops on Runpod.
> 
> I suppose I will have to look for some other solution to train after just having used your stuff for two years. I'm very unhappy with this.

You're wasting your time with this guy. His repo is shit, and he doesn't care.

**Comment by jockisen** [comment]
> > @TheLastBen
> > <img alt="Skärmavbild 2024-07-23 kl 00 11 16" width="958" src="https://private-user-images.githubusercontent.com/112794049/351127119-ee7d0a16-1e6f-455c-9fc3-927d9544a547.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE3MDAxNDAsIm5iZiI6MTcyMTY5OTg0MCwicGF0aCI6Ii8xMTI3OTQwNDkvMzUxMTI3MTE5LWVlN2QwYTE2LTFlNmYtNDU1Yy05ZmMzLTkyN2Q5NTQ0YTU0Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyM1QwMTU3MjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MmRhOGVjMmVkZGQ5OGMyMjRmZTU1Yjk4ZjMxNzg4ZjMwNmMxM2EzMmE3Mjc1MTA2NzNlYjE4OGRiNzAwZDAxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.W1BZ-YY-A8XpzgIfYhznKLD2mKc8TSxuGHTFFjGPjfk">
> > this is how slow it works on the cheapest ($ 1.10/h) 24 gb vram gpu on Paperspace with the exact same settings I've used on Runpod. this is for a 30 image dataset. wouldve taken me like 15 minutes tops on Runpod.
> > I suppose I will have to look for some other solution to train after just having used your stuff for two years. I'm very unhappy with this.
> 
> You're wasting your time with this guy. His repo is shit, and he doesn't care.

The problem is that I rely on the trainer for work and have been using his stuff for a long time. It’s not so easy to just make the transition but I suppose I must.

**Comment by qudabear** [comment]
> > > @TheLastBen
> > > <img alt="Skärmavbild 2024-07-23 kl 00 11 16" width="958" src="https://private-user-images.githubusercontent.com/112794049/351127119-ee7d0a16-1e6f-455c-9fc3-927d9544a547.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE3MDAxNDAsIm5iZiI6MTcyMTY5OTg0MCwicGF0aCI6Ii8xMTI3OTQwNDkvMzUxMTI3MTE5LWVlN2QwYTE2LTFlNmYtNDU1Yy05ZmMzLTkyN2Q5NTQ0YTU0Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyM1QwMTU3MjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MmRhOGVjMmVkZGQ5OGMyMjRmZTU1Yjk4ZjMxNzg4ZjMwNmMxM2EzMmE3Mjc1MTA2NzNlYjE4OGRiNzAwZDAxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.W1BZ-YY-A8XpzgIfYhznKLD2mKc8TSxuGHTFFjGPjfk">
> > > this is how slow it works on the cheapest ($ 1.10/h) 24 gb vram gpu on Paperspace with the exact same settings I've used on Runpod. this is for a 30 image dataset. wouldve taken me like 15 minutes tops on Runpod.
> > > I suppose I will have to look for some other solution to train after just having used your stuff for two years. I'm very unhappy with this.
> > 
> > 
> > You're wasting your time with this guy. His repo is shit, and he doesn't care.
> 
> The problem is that I rely on the trainer for work and have been using his stuff for a long time. It’s not so easy to just make the transition but I suppose I must.

Paperspace is garbage. You're better off switching to a different repo. Paperspace approves GPU uses on a case by case basis and they ask for all of your personal information, and they also take 10 dollars just to verify your card is good. Garbage company.

**Comment by jockisen** [comment]
> > > > @TheLastBen
> > > > <img alt="Skärmavbild 2024-07-23 kl 00 11 16" width="958" src="https://private-user-images.githubusercontent.com/112794049/351127119-ee7d0a16-1e6f-455c-9fc3-927d9544a547.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE3MDAxNDAsIm5iZiI6MTcyMTY5OTg0MCwicGF0aCI6Ii8xMTI3OTQwNDkvMzUxMTI3MTE5LWVlN2QwYTE2LTFlNmYtNDU1Yy05ZmMzLTkyN2Q5NTQ0YTU0Ny5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyM1QwMTU3MjBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MmRhOGVjMmVkZGQ5OGMyMjRmZTU1Yjk4ZjMxNzg4ZjMwNmMxM2EzMmE3Mjc1MTA2NzNlYjE4OGRiNzAwZDAxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.W1BZ-YY-A8XpzgIfYhznKLD2mKc8TSxuGHTFFjGPjfk">
> > > > this is how slow it works on the cheapest ($ 1.10/h) 24 gb vram gpu on Paperspace with the exact same settings I've used on Runpod. this is for a 30 image dataset. wouldve taken me like 15 minutes tops on Runpod.
> > > > I suppose I will have to look for some other solution to train after just having used your stuff for two years. I'm very unhappy with this.
> > > 
> > > 
> > > You're wasting your time with this guy. His repo is shit, and he doesn't care.
> > 
> > 
> > The problem is that I rely on the trainer for work and have been using his stuff for a long time. It’s not so easy to just make the transition but I suppose I must.
> 
> Paperspace is garbage. You're better off switching to a different repo. Paperspace approves GPU uses on a case by case basis and they ask for all of your personal information, and they also take 10 dollars just to verify your card is good. Garbage company.

I suppose that’s the end of using @TheLastBen trainer then. At least for now.

**Comment by qudabear** [comment]
Just an update for you all. I've talked with Runpod and Ben is full of shit. They didn't "discontinue support" - he intentionally broke the pod because he demanded money and they refused. They've given me a credit for the money I've lost because of this BS, and they are working on helping people with a different option for training, so communicate with them directly about your options! As for Ben, he's shady, slimey, straight up lied, and I won't be using anything from him ever again.

**Comment by jockisen** [comment]
> Just an update for you all. I've talked with Runpod and Ben is full of shit. They didn't "discontinue support" - he intentionally broke the pod because he demanded money and they refused. They've given me a credit for the money I've lost because of this BS, and they are working on helping people with a different option for training, so communicate with them directly about your options! As for Ben, he's shady, slimey, straight up lied, and I won't be using anything from him ever again.

Wow. Would you mind connect on Discord? I’m joachimsallstrom on there.

**Comment by TheLastBen** [comment]
@qudabear I don't think you're mature enough to understand the meaning of a contract, the whole notebooks were built based on a contract, they have the liberty to stop paying if I stop maintaining the notebooks and I'm in liberty to discontinue the notebooks if they stop paying, I'm not in liberty to share the contract, otherwise I would've shared it.

**Comment by qudabear** [comment]
> @qudabear I don't think you're mature enough to understand the meaning of a contract, the whole notebooks were built based on a contract, they have the liberty to stop paying if I stop maintaining the notebooks and I'm in liberty to discontinue the notebooks if they stop paying, I'm not in liberty to share the contract, otherwise I would've shared it.

You've got to be joking. You are calling ME immature? You're the clown that broke a notebook and ruined a service for hundreds of people that depended on your depo, purely because you were mad you couldn't profit off of it from Runpod. If you were actually trying to help the community, you would have realized how losing the notebook would affect people, and you would have left it on Runpod for free, but no, like a child, you had to be spiteful, straight up LIE and say that they "discontinued support", and then break the notebook on purpose. You're a clown, and a child. Calling me immature is hilarious.

**Comment by TheLastBen** [comment]
@qudabear I'm going to try and stay civil with you, this repo is for colab notebooks and the notebooks are still here and will always stay here. I had an agreement with runpod to build notebooks for them, get them hundreds if not thousands of clients, promote their website and redirect clients to them, in return for 25$/day as long as I do all the things I mentioned, that was the agreement, then later, their marketing department decided to stop paying because they got all the marketing they needed, hoping I'll still work for them for free, they were wrong, they should know better, they also cut services when clients stop paying.

They decided to put the blame on me, a freelancer who promotes their website and who got them thousands of users for 25$/day, they are responsible for cutting the service and they should quickly find someone to build a Stable Diffusion template for them because they will lose all those thousands of users I got for them.

**Comment by bartman081523** [comment]
https://hub.docker.com/layers/runpod/stable-diffusion/fast-stable-diffusion-2.4.0/images/sha256-c3f7815767b580ac7d2994e46e461ba3cf85b1610eff1301fb341d27451c7033?context=explore

**Comment by HelpANewbie** [comment]
Hi @TheLastBen  - Which platform do you suggest to use for the same Fast SD since Runpod is not supporting now ?

**Comment by TheLastBen** [comment]
@HelpANewbie use Paperspace, link in the main page of the repo

**Comment by qudabear** [comment]
> @qudabear I'm going to try and stay civil with you, this repo is for colab notebooks and the notebooks are still here and will always stay here. I had an agreement with runpod to build notebooks for them, get them hundreds if not thousands of clients, promote their website and redirect clients to them, in return for 25$/day as long as I do all the things I mentioned, that was the agreement, then later, their marketing department decided to stop paying because they got all the marketing they needed, hoping I'll still work for them for free, they were wrong, they should know better, they also cut services when clients stop paying.
> 
> They decided to put the blame on me, a freelancer who promotes their website and who got them thousands of users for 25$/day, they are responsible for cutting the service and they should quickly find someone to build a Stable Diffusion template for them because they will lose all those thousands of users I got for them.

Hey! He provides actual info! Maybe if you had done that ages ago, people would have been able to hear your side of the story and actually sympathize with you a little. Instead, the tool we've all been using went down over a payment dispute, you intentionally broke the login, vaguely stated that Runpod had "ended support" (not true), and then, as you always do, didn't respond to direct questions about anything until now. Look, I appreciate all of your work, and I appreciate the struggle of being a freelancer. I do love your repo and think it has amazing potential, but you've been completely ignore your community when we ask direct, basic questions, and given that, according to you, Runpod completely lied to us, which I am definitely willing to believe, we had no other alternative but to assume you had fucked us all. Thank you for telling us your side. I apologize for assuming, but had little to go on, and I'm glad you've cleared it up. As for Paperspace, they are absolutely AWFUL, and I will never use a Google product so: is there any other way we can use your repo. Also, I'd highly recommend just making a Patreon or Ko-fi and engaging with us regularly, and making a local version or your own hosted version of your notebook. I'd happily pay to support you and you'd make far, far more than 25 per month, guaranteed!

**Comment by ashleykleynhans** [comment]
@qudabear  it was $25 per day, so around $750 per month, not $25 per month that RunPod were paying.

**Comment by ashleykleynhans** [comment]
@TheLastBen I am completely on your side.

RunPod keeps banning me from their Discord server after all the work I have put into building literally dozens of templates for their platform.  They have obviously grown too big for their boots since they received a significant round of investment, so I will also no longer be supporting the RunPod platform until they change their attitude towards community members that put in so much hard work to making their platform a success.

**Comment by jockisen** [comment]
> @TheLastBen I am completely on your side.
> 
> RunPod keeps banning me from their Discord server after all the work I have put into building literally dozens of templates for their platform. They have obviously grown too big for their boots since they received a significant round of investment, so I will also no longer be supporting the RunPod platform until they change their attitude towards community members that put in so much hard work to making their platform a success.

So what alternative beside the horrible Paperspace do you suggest?

**Comment by ibxx0402** [comment]
Didn't this also happen like a while ago  with paper space? I remember switching to run pod because of paper space not working anymore. And not this doesn't work anymore, does google collab still work or is that also out of commission? its kinda hard to follow where to run this, for users that uses it once in a blue moon, as the service changes so often. 

Also remember to delete the notebook from paper space after you have used them as they will bill for the storage.

**Comment by jockisen** [comment]
Just go and use a trainer for Flux instead. It’s so much better anyway. 13 aug. 2024 kl. 12:13 skrev Ib Leminen Mohr Nielsen ***@***.***>:﻿
Didn't this also happen like a while ago  with paper space? I remember switching to run pod because of paper space not working anymore. And not this doesn't work anymore, does google collab still work or is that also out of commission? its kinda hard to follow where to run this, for users that uses it once in a blue moon, as the service changes so often.
Also remember to delete the notebook from paper space after you have used them as they will bill for the storage.

**Comment by Ibxx0403** [comment]
@jockisen i sadly use a mac and draw things which i dont think uses flux, so i don't think this would solve the issue.

**Comment by krmorex** [comment]
This shit looks like a Venezuelan TV series... solution. a new and free notebook on runpod. who does it? end

**Comment by lwc-alex** [comment]
I am just amused to see people getting mad because they don't get what the want for free.

**Comment by exliko** [comment]
> I am just amused to see people getting mad because they don't get what the want for free.

Well, I'm not call it all free. We pay deposit in runpod and we want to use Ben's Pod. Since it's gone, our deposit left is "useless" for now and for my case, runpod won't refund it.
